src/Crypto/core-crypto-tests.asd

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
@dbmcclain The convention I am trying to follow with defining test systems is to use the singular `-test` (instead of `-tests` for the package and ASDF systems.  While this may seem very "nitpicky", it would be nice to all follow this convention, as I normalized all other test systems to this convention.

Reply:
okay FB